### PR TITLE
#64 - Updated implementation of one finger tap

### PR DESCRIPTION
**Pull request by jrdike**
.

### CODE REVIEW

**Comment by xorbit3024** [comment]
Please add unit tests to verify changes.

File: gesturelibrary/include/single/singleFingerTap.h [hunk 1]
```diff
@@ -11,6 +11,10 @@
 #define TAP_DISTANCE 100
```
Comment (xorbit3024) [inline gesturelibrary/include/single/singleFingerTap.h:11]: Please use params existing or define new params in gestureparams.h

File: gesturelibrary/include/single/singleFingerTap.h [hunk 2]
```diff
@@ -11,6 +11,10 @@
...
 #endif
 
+#ifndef TAP_TIME
+#define TAP_TIME 7
```
Comment (xorbit3024) [inline gesturelibrary/include/single/singleFingerTap.h:15]: Please use params existing or define new params in gestureparams.h

Comment (jrdike) [reply gesturelibrary/include/single/singleFingerTap.h]: Fixed

**Review by xorbit3024 — APPROVED** [review]

**Pull request merged**